recipe/meta.yaml

```
@@ -27,7 +29,7 @@ requirements:
     - wheel
     - mkl-devel  {{ mkl }}
     - cython 3
-    - numpy-base {{ numpy }}
+    - numpy-base >=1.16
```

```suggestion
    - numpy-base {{ numpy }}
```
mkl_random uses the numpy C API. We have to sue the pinning from aggregate's CBC file.